backend/endpoints/messages.py

```diff
@@ -186,12 +186,13 @@ async def update_message(
             detail="You are not authorized to edit this message",
         )
 
-    message.update(payload)
-    message["edited"] = True
+    payload["edited"] = True
     edited_message = await DB.update(
-        MESSAGE_COLLECTION, document_id=message_id, data=message
+        MESSAGE_COLLECTION, document_id=message_id, data=payload
     )
 
+    message.update(payload)
```

Comment:
I think this should come before payload["edited"] = True

Reply:
@pauline-banye, if we update the message object before setting the edit field to True in the payload, the edit field will still be False in the response.

Reply:
Ohhhh I forgot about the edited true flag 🤦‍♀️. 
Yeah it's needed beneath the edited true but why all the way underneath the await db.update?

Reply:
Good point.
It's a common practice to declare/initialize variables as close to their usage as possible.
It's not a fixed rule though.

Reply:
Yeah for "readability" tho but it works and all the tests pass. Great job on the fix boss👍🏻